src/Form/ArticleFormType.php

```diff
@@ -0,0 +1,44 @@
+<?php
+namespace App\Form;
+
+use App\Entity\Author;
+use App\Entity\Article;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\Form\CallbackTransformer;
+use Symfony\Component\Form\Extension\Core\Type\TextType;
+use Symfony\Component\Form\Extension\Core\Type\TextareaType;
+use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
+use Symfony\Bridge\Doctrine\Form\Type\EntityType;
+use Symfony\Component\Form\Extension\Core\Type\SubmitType;
+
+class ArticleFormType extends AbstractType
+{
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        $builder
+
+            ->add('category', ChoiceType::class, array(
+                'choices'  => array(
+                    'Обзор' => Article::CATEGORY_OVERVIEW,
+                    'Объявление' => Article::CATEGORY_ANNOUNCEMENT,
+                    'Статья' => Article::CATEGORY_ARTICLE
+                ),
+//                'expanded' => true,
```

Comment:
@daria-sieroshtan 
Pls remove commented code
I agree with you that select is better than radios